inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/AutoPush2HiveExample.java

```diff
@@ -29,18 +29,20 @@
 import org.apache.inlong.manager.common.pojo.stream.InlongStreamInfo;
 import org.apache.inlong.manager.common.pojo.stream.StreamField;
 import org.apache.shiro.util.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
 /**
  * Test class for auto push to hive.
  */
-public class AutoPush2HiveExample extends BaseExample {
+@Disabled
+class AutoPush2HiveExample extends BaseExample {
 
     @Test
```

Comment:
If this is not the unit test class, suggest removing all `@Test` annotations.

Reply:
This port has been used

Reply:
OK, good idea.